lib/orbit/authentication/user.ex

```diff
@@ -20,7 +20,7 @@ defmodule Orbit.Authentication.User do
     field(:email, :string)
     field(:permissions, {:array, UserPermission})
     field(:groups, {:array, :string}, virtual: true)
-    timestamps()
+    timestamps(type: :utc_datetime)
```

Comment:
An alternative way to configure this is to use the `@timestamps_opts ` [schema attribute](https://hexdocs.pm/ecto/Ecto.Schema.html#module-schema-attributes), which is meant to be called after `use Ecto.Schema` but before the call to `schema`.

The only advantage I can see to doing it that way is we could actually write our own wrapper macro, something like `use Orbit.Ecto.Schema`, which combines the two so we wouldn't need to repeat this for every call to `timestamps`. But I didn't go that route because I think maybe that obscures what's happening unless someone knows to go look inside that macro.